module1/task1/models/person.go

```diff
@@ -13,7 +16,7 @@ type IPerson interface {
 	ChangeName(name string) string
 }
 
-func NewPerson(name string, age uint8, educationalInstitution string) IPerson {
+func NewPerson(name string, age uint8, educationalInstitution string) *Person {
```

Comment:
In your code I saw some things like `*NewPerson`, if you deref it pls just return the structure, not a pointer to it.
```suggestion
func NewPerson(name string, age uint8, educationalInstitution string) Person {
```